pymesync.py

```diff
@@ -788,6 +807,24 @@ def __create_or_update(self, parameters, identifier,
             # Request error
             return [{self.error: e}]
 
+    def __interpret_times(self, duration):
```

Comment:
Can we call this something more descriptive, like ``__convert_duration_string`` or ``__duration_to_seconds``? What do you think?